bee-ternary/CHANGELOG.md

```diff
@@ -19,6 +19,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+<!-- ## Unreleased - YYYY-MM-DD
+
+### Added
+
+- Support for arbitrary trit to numeric type conversion
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security -->
+
```

Comment:
Just do a section like the previous 2 so that we can release `0.3.0-alpha` today :)

Reply:
With today's date

Reply:
Ok